tests/components/homeassistant/test_init.py

```diff
@@ -371,3 +371,18 @@ def test_check_config(self, mock_check, mock_stop):
 
     assert len(mock_update.mock_calls) == 1
     assert mock_update.mock_calls[0][1][1] == 'light.kitchen'
+
+
+async def test_setting_location(hass):
+    """Test setting the location."""
+    await async_setup_component(hass, 'homeassistant', {})
+    events = async_capture_events(hass, EVENT_CORE_CONFIG_UPDATE)
+    assert hass.config.latitude != 10
```

Comment:
I'm confused on why you're asserting it's not 10 and 20... shouldn't this either assert it's some other value than what is being set OR assert that it's NOT 30 and 40?

Reply:
You are right, I should have done not 30 and 40, just to make sure my update works.